app/components/SectionTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Text } from 'react-native'
+import tailwind from 'tailwind-rn'
+
+export function SectionTitle ({ text, testID }: { text: string, testID: string }): JSX.Element {
+  return (
+    <Text
+      testID={testID}
+      style={[tailwind('p-4 text-xs text-gray-500 font-bold mt-2')]}
+    >
+      {
+        text
```

Comment:
minor suggestion: maybe you could have this as a {props.children} instead?

Reply:
ah yeah good idea. If we have a section title with info symbol or tooltip, might be useful. I'll do it on next PR